libcontainer/container_linux.go

```diff
@@ -1249,7 +1254,7 @@ func (c *linuxContainer) orderNamespacePaths(namespaces map[configs.NamespaceTyp
 			if strings.ContainsRune(p, ',') {
 				return nil, newSystemError(fmt.Errorf("invalid path %s", p))
 			}
-			paths = append(paths, p)
+			paths = append(paths, fmt.Sprintf("%s:%s", configs.NsName(nsType), p))
```

Comment:
It doesn't fix any current bugs, but it does allow us to make sure that we don't accidentally add any bugs with regards to the ordering of namespace paths (which would cause very, very odd issues that probably wouldn't be easy to diagnose).